src/ecs/explorer/ecsContainerNode.ts

```diff
@@ -32,10 +31,27 @@ export class EcsContainerNode extends AWSTreeNodeBase {
     }
 
     public listTasks() {
-        return this.ecs.listTasks(this.clusterArn, this.serviceName)
+        return this.ecs.listTasks({ cluster: this.clusterArn, serviceName: this.serviceName })
     }
 
     public describeTasks(tasks: string[]) {
         return this.ecs.describeTasks(this.clusterArn, tasks)
     }
+
+    private async hasRunningTasks(): Promise<boolean> {
+        return (
+            (
+                await this.ecs.listTasks({
+                    cluster: this.clusterArn,
+                    serviceName: this.serviceName,
+                    desiredStatus: TASK_STATUS_RUNNING,
+                    maxResults: 1,
+                })
+            ).length > 0
+        )
+    }
+
+    public async updateRunningTasks(): Promise<void> {
+        this.description = (await this.hasRunningTasks()) ? false : `[no running tasks]`
```

Comment:
Does `false` get rendered in some way/does the compiler yell at you for setting that? Does `no running tasks` need to be localized, and also, unnecessary backticks (unless you want to add something programmatically to the string)

Reply:
`false` is the default for the node description and will not cause any issues. Localized the text.